scripts/check-workflows.sh

```diff
@@ -17,7 +17,7 @@ if [ ! -d transforms ]; then
     echo Please run this script from the top of the repository
     exit 1
 fi
-KFP_BLACK_LIST="doc_chunk pdf2parquet pii_redactor text_encoder license_select repo_level_ordering header_cleanser fdedup hap rep_removal gneissweb_classification"
+KFP_BLACK_LIST="doc_chunk pdf2parquet pii_redactor text_encoder license_select repo_level_ordering comment_cleanser header_cleanser fdedup hap rep_removal gneissweb_classification"
```

Comment:
@shivdeep-singh-ibm @matouma This is a new transform. I don't have understanding of this black list. when would we need to a transform to black list?